storey/sources.py

```diff
@@ -1012,3 +1012,82 @@ def _init(self):
             else:
                 df = pandas.read_parquet(path, columns=self._columns, storage_options=self._storage_options)
             self._dfs.append(df)
+
+
+class SQLSource(_IterableSource, WithUUID):
+    """Use SQL table as input source for a flow.
+
+
+    :parameter key_field: the primary key of the table.
+    :parameter time_field: column to be used as time for events.
+    :parameter id_field: column to be used as ID for events.
+    :parameter db_name: url string connection to sql database.
+    :parameter table_name: the name of the table to access, from the current database
+    """
+
+    def __init__(
+        self,
+        db_path: str,
+        table_name: str,
+        key_field: Optional[Union[None, str, List[str]]] = None,
```

Comment:
```suggestion
        key_field: Union[None, str, List[str]] = None,
```
Optional[X] == Union[None, X]